plugins/modules/one_image.py

```diff
@@ -114,300 +107,432 @@
 id:
     description: image id
     type: int
-    returned: success
+    returned: when O(state=present, state=cloned, state=renamed)
     sample: 153
 name:
     description: image name
     type: str
-    returned: success
+    returned: when O(state=present, state=cloned, state=renamed)
     sample: app1
 group_id:
     description: image's group id
     type: int
-    returned: success
+    returned: when O(state=present, state=cloned, state=renamed)
     sample: 1
 group_name:
     description: image's group name
     type: str
-    returned: success
+    returned: when O(state=present, state=cloned, state=renamed)
     sample: one-users
 owner_id:
     description: image's owner id
     type: int
-    returned: success
+    returned: when O(state=present, state=cloned, state=renamed)
     sample: 143
 owner_name:
     description: image's owner name
     type: str
-    returned: success
+    returned: when O(state=present, state=cloned, state=renamed)
     sample: ansible-test
 state:
     description: state of image instance
     type: str
-    returned: success
+    returned: when O(state=present, state=cloned, state=renamed)
     sample: READY
 used:
     description: is image in use
     type: bool
-    returned: success
+    returned: when O(state=present, state=cloned, state=renamed)
     sample: true
 running_vms:
     description: count of running vms that use this image
     type: int
-    returned: success
+    returned: when O(state=present, state=cloned, state=renamed)
     sample: 7
+permissions:
+    description: The image's permissions.
```

Comment:
Is this a new return value, or hasn't it been documented so far? If it's a new return value, you need to add `version_added: 9.5.0` to it.